analyses/molecular-subtyping-EPN/01-assign-disease-group.R

```diff
@@ -30,35 +30,56 @@ EP = pbta_histologies %>%
 
 # filter for RNA samples
 EP_rnaseq_samples = EP %>%
-  filter(experimental_strategy == "RNA-Seq") %>%
+  filter(experimental_strategy == "RNA-Seq" | experimental_strategy == "Targeted Sequencing"
+         & (!is.na(RNA_library) & experimental_strategy == "Targeted Sequencing")) %>%
   dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, primary_site, CNS_region) %>%
   dplyr::rename("Kids_First_Biospecimen_ID_RNA" = "Kids_First_Biospecimen_ID")
 
 # filter for DNA samples 
 WGS_dnaseqsamples = EP %>%
-  filter(experimental_strategy == "WGS") %>%
+  filter(experimental_strategy == "WGS" | experimental_strategy == "WXS" | experimental_strategy == "Targeted Sequencing",
+         is.na(RNA_library)) %>%
   dplyr::select(Kids_First_Biospecimen_ID, Kids_First_Participant_ID, sample_id, primary_site, CNS_region) %>%
   dplyr::rename("Kids_First_Biospecimen_ID_DNA" = "Kids_First_Biospecimen_ID")
 
-# sample_id is common between both  dataframes and also unique between RNA and DNA. 
+# filter for Methyl samples 
+methy_samples = EP %>%
+  dplyr::filter(experimental_strategy == "Methylation", 
+                (grepl("EPN_", dkfz_v12_methylation_subclass) & dkfz_v12_methylation_subclass_score >= 0.8)) %>%
+  dplyr::mutate(molecular_subtype_methyl = case_when(grepl("EPN_PFA", dkfz_v12_methylation_subclass) ~ "EPN, PF A",
+                                                     grepl("EPN_PFB", dkfz_v12_methylation_subclass) ~ "EPN, PF B",
+                                                     grepl("EPN_ST_ZFTA", dkfz_v12_methylation_subclass) ~ "EPN, ST ZFTA",
+                                                     dkfz_v12_methylation_subclass == "EPN_MPE" ~ "EPN, MPE",
+                                                     dkfz_v12_methylation_subclass == "EPN_PF_SE" ~ "EPN, PF SE",
+                                                     dkfz_v12_methylation_subclass == "EPN_SPINE" ~ "EPN, SP",
+                                                     dkfz_v12_methylation_subclass == "EPN_SPINE_MYCN" ~ "EPN, SP-MYCN",
+                                                     dkfz_v12_methylation_subclass == "EPN_SPINE_SE_A" ~ "EPN, SP-SE A",
+                                                     dkfz_v12_methylation_subclass == "EPN_SPINE_SE_B" ~ "EPN, SP-SE B",
```

Comment:
```suggestion
                                                     dkfz_v12_methylation_subclass == "EPN_SPINE_SE_A" ~ "EPN, SP-SE",
                                                     dkfz_v12_methylation_subclass == "EPN_SPINE_SE_B" ~ "EPN, SP-SE",
```

minor update request here